velox/substrait/TypeUtils.cpp

```diff
@@ -17,45 +17,87 @@
 #include "velox/type/Type.h"
 
 namespace facebook::velox::substrait {
-
-bool isString(const TypePtr& type) {
-  switch (type->kind()) {
-    case TypeKind::VARCHAR:
-    case TypeKind::VARBINARY:
-      return true;
-    default:
-      break;
+namespace {
+std::vector<std::string> getRowTypesFromCompoundName(
```

Comment:
this could also be refactored using a super simple bison grammar to parse these types (since we already have a dependency on bison/flex). Maybe create an Issue, if someone is willing to pick it up?

Reply:
Added a TODO.